sodetlib/det_config.py

```diff
@@ -643,6 +651,8 @@ def get_smurf_control(self, offline=False, epics_root=None,
                 make_logfile=make_logfile, data_path_id=smurfpub_id,
                 **pysmurf_kwargs)
         self.S = S
+        # Lets just stash this in pysmurf...
+        S._sodetlib_cfg = self
```

Comment:
This is used in the stream function update to get the default downsample factor and mode. I decided to put this in the pysmurf instance so that we didn't have to add `cfg` as an argument to the `stream_g3_on` function, which is a large interface change.